tests/XRootD/scitokens.sh

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env bash
+
+function setup_scitokens() {
+	require_commands curl
+	if [ -z "$HOSTNAME" ]; then
+		HOSTNAME=`hostname`
+	fi
+
+	cat > "$PWD/scitokens/authdb" << EOF
+u * /issuer lr
+EOF
+	cat > "$PWD/scitokens/scitokens.cfg" <<EOF
+[Global]
+audience = https://$HOSTNAME:8095
+
+[Issuer test]
+issuer = https://$HOSTNAME:8095/issuer/one
+base_path = /protected
+EOF
+
+	# Setup the issuer and protected directory
+	ln -sf "$PWD/../issuer/export" "$PWD/scitokens/xrootd/issuer"
+	mkdir -p "$PWD/scitokens/xrootd/protected"
```

Comment:
It's cleaner to remove `$PWD/` from paths and use relative paths if it doesn't break anything. For example:
```sh
mkdir -p scitokens/xrootd/protected
```

Reply:
Took a quick review through the code and did this in most places (the prior `ln` line needs the absolute version so I left it there).